debian/control

```diff
@@ -0,0 +1,58 @@
+Source: nymea-plugins-zwave
+Section: utils
+Priority: options
+Maintainer: nymea GmbH <developer@nymea.io>
+Build-depends: debhelper (>= 0.0.0),
+               libnymea-dev (>= 0.26~),
+               nymea-dev-tools:native,
+               pkg-config,
+               qtbase5-dev,
+Standards-Version: 3.9.3
+
+
+Package: nymea-plugin-zwave-generic
+Architecture: any
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+Description: nymea integration plugin for ZigBee Spec compliant devices
```

Comment:
`s/ZigBee/ZWave/` and update the description since it does not seem to match the Zwave even replacing the ZigBee